spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -174,6 +175,33 @@ def _find_nonsys32_bash_exe():
     return shutil.which('bash', path=nonsys32_paths)
 
 
+def _filter_directories(dir_list, include=None, include_list=None, exclude=None, exclude_list=None):
+    # Handle inclusion lists
+    if include is not None:
+        dir_list = [f for f in dir_list if re.search(include, f) is not None]
+
+    if include_list is not None:
+        # TODO decide if we should warn users if one of their inclusions isn't around
```

Comment:
We might want to address this existing `# TODO` here?

----

My vote would be to remove the TODO without warning...

I'm not sure how much a warning will help, because if it's just a warning, then the processing will continue as normal, and I suspect that the user won't catch it due to the text that will be printed underneath it.

In other words, it seems like the user will be just as likely to notice the error without the message, once they realize that X subject wasn't processed.

Reply:
I agree, an outright error would be upsetting, and a warning would get lost in the sea of output. And the argument descriptions are explicitly worded to indicate that these arguments are filters.